### PR TITLE
Resolving minor bug where company logo overlaped next column

### PR DESCRIPTION
**Pull request by chu4**
Company logo overlapped next column element for phone view
Old
image
New
image

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -18,4 +18,9 @@
...
     text-decoration: none;
   }
-}
\ No newline at end of file
+}
+
+.company-logo {
+  max-height: 100%;
+  max-width: 100%;
+  opacity: 1; }
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/custom.scss]: move "}" to next line and merge PR

Comment (igorkasyanchuk) [reply app/assets/stylesheets/custom.scss]: and maybe actuallt try to use bootstrap image css class "img-responsive" or something like that

Comment (chu4) [reply app/assets/stylesheets/custom.scss]: I tried using that class but it didn't work, somehow it is not accesible or something at least it doesn't show up styles on the right down corner
image

Comment (igorkasyanchuk) [reply app/assets/stylesheets/custom.scss]: ok, approved

**Pull request merged**